core/src/i18n/pl.yaml

```
@@ -100,29 +116,32 @@ namespaces:
     title: Przegląd
     workloads:
       deployments: $t(deployments.title)
-      error: 'Błąd podczas ładowania {{title}} danych z powodu: {{message}}'
+      error: 'Błąd podczas ładowania danych dla {{title}} z powodu: {{message}}'
       pods: $t(pods.title)
       title: Zdrowe zasoby
   title: Przestrzenie nazw
   tooltips:
     container-memory-limit: Zdefiniuj ograniczenia pamięci dla poszczególnych kontenerów w swojej przestrzeni nazw.
-    create: Wybierz tę opcję, aby wyłączyć istio, aby pośredniczyć w całej komunikacji między zasobnikami w Twojej przestrzeni nazw.
-    memory-examples: Użyj zwykłej wartości w bajtach (128974848) lub odpowiedników przyrostków (takich jak 129e6, 129M, 123Mi).
+    create: Wybierz tę opcję, aby wyłączyć istio, aby pośredniczyć w całej komunikacji między Podami w Twojej przestrzeni nazw.
```

And, I guess, if we don't capitalize the terms, then `Pods` should also be lowercased in Polish. 
```suggestion
    create: Wybierz tę opcję, aby wyłączyć istio, aby pośredniczyć w całej komunikacji między podami w Twojej przestrzeni nazw.
```